Tests/bytesTests/Strings+LocalizableTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Strings+LocalizableTests.swift
+//  
+//
+//  Created by Thorsten Stark on 30.05.22.
+//
+
+import XCTest
+
+class Strings_LocalizableTests: XCTestCase {
+
+    func testSimpleLocalizedWithBundle() throws {
+        XCTAssertEqual("test.key".localized(bundle: .module), "it works!")
+    }
+    
+    func testSimpleLocalizedWithCorrectKey() throws {
```

Comment:
This test basically tests custom localization tables, right? Shouldn't this than be reflected in the test name?

Reply:
Your absolutely right. Changed the name and also added 3 more tests.